src/utils/dataFetch.js

```diff
@@ -2,10 +2,10 @@ import fetch from "isomorphic-fetch";
 
 const API_URL = "https://api.amfoss.in/";
 
-export default ({ query, variables }) => {
+export default ({ query, userInfo }) => {
```

Comment:
```suggestion
export default ({ query, variables }) => {
```

I would prefer it to be named as variables. We use it for different cases.

Reply:
Kinda weird, but still.